samples/excel/tables/create-table.yaml

```diff
@@ -0,0 +1,94 @@
+id: ''
+gist: ''
+source: EXCEL
+author: Microsoft
+name: Create a table
+description: Creates a table with four columns and seven rows.
+script:
+  content: |+
+    $('#create-table').click(createTable);
+
+    /* Create a new table with some sample data */
+    function createTable() {
+        Excel.run(async (context) => {
+            let sheet: Excel.Worksheet;
+            try {
+                // Queue a command to add a new sheet
+                sheet = context.workbook.worksheets.add();
```

Comment:
Pls move this code to a setup function and reuse the pattern in https://github.com/WrathOfZombies/samples/blob/master/samples/excel/range/set-get-values.yaml with the resetWorksheet helper function.